src/memphis.ts

```diff
@@ -611,7 +611,7 @@ class Memphis {
   async station({
     name,
     retentionType = retentionTypes.MAX_MESSAGE_AGE_SECONDS,
-    retentionValue = 604800,
+    retentionValue = 3600,
```

Comment:
run npm run build and add the build files, and please open an issue in the .net SDK